tools/allowed-large-files

```diff
@@ -48,6 +48,7 @@ scripts/ci/lib.sh
 sensor/tests/data/policies.json
 sensor/upgrader/preflight/testdata/k8s-1.22.json.gz
 tests/performance/load/package-lock.json
+ui/.yarn-dist/yarn-*.tgz
```

Comment:
I am confused here. You added the `ui/.yarn-dist/yarn-1.22.21.tgz` file to the repository, but this line will ignore changes to it (which would be ok) or new versions. 

How about removing the line or adding a comment to the `prepare` target and README to ask for `git add --force ui/.yarn-dist/yarn-<new version>.tgz` ?

Reply:
This isn't `.gitignore`, it's `allowed-large-files`, part of our linter that reminds not to check in large files to the repo. I think, the exclusion is warranted in this case because the usage is similar how we distribute gradle wrapper https://github.com/stackrox/stackrox/tree/master/qa-tests-backend/gradle/wrapper